tensorboard/plugins/interactive_inference/tf_interactive_inference_dashboard/tf-interactive-inference-dashboard.html

```diff
@@ -158,6 +158,12 @@
         padding-right: 5px;
       }
 
+      .pd-no-features-text {
+        font-size: 18px;
+        color: #777;
```

Comment:
stylistic nit: css property names should be ordered alphabetically a-z here and below